src/style.css

```diff
@@ -1,3 +1,8 @@
 h1, p {
   font-family: Lato;
-}
\ No newline at end of file
+}
+
+.buttons-line {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction` is `row` by default, so you can omit this rule

Reply:
Actually, I needed to add it, because default behaviour was different.